src/components/FooterLogo.spec.jsx

```diff
@@ -30,8 +31,7 @@ describe('FooterLogo', () => {
         }
       }
     })
-    const component = shallow(<FooterLogo client={mockClient} />)
-    await new Promise(resolve => setImmediate(resolve)) // await the didMount
-    expect(component.getElement()).toMatchSnapshot()
+    render(<FooterLogo client={mockClient} />)
+    expect((await screen.findAllByRole('img')).length).toBeGreaterThanOrEqual(1)
```

Comment:
```
await render(<FooterLogo client={mockClient} />)
expect((screen.getAllByRole('img')).length).toEqual(2)
```
1) **find => get** because, if ever the test failed, we won't have to wait 1 second
2) **toBeGreaterThanOrEqual => toEqual**, to show the exact number of logo. (what if I duplicate the <img/> tag in the component?) This makes test more robust

Reply:
Couldn't make it work with get.
Updated gtThn to toEqual() though